backend/apps/ifc_validation/tasks.py

```diff
@@ -393,18 +409,31 @@ def parse_info_subtask(self, prev_result, id, file_name, *args, **kwargs):
 
                 # MVD
                 try:
-                    model.mvd = ifc_file.header.file_description.description[0].split(" ", 1)[1][1:-1]
+                    if using_purepythonparser:
+                        # ifcopenshell.simple_spf returns instance attributes as tuples,
+                        # also header information.
+                        desc = ifc_file.header.file_description[0][0]
+                    else:
+                        desc = ifc_file.header.file_description.description[0]
+                    # @todo implement based on grammar in header agreement PDF
```

Comment:
Perhaps this will be covered by IVS-179 / the [header policy checks ](https://github.com/buildingSMART/IFC4.x-IF/tree/header-policy/docs/IFC-file-header) ? 
There's at least some overlap.